modules/storages/config/locales/en.yml

```diff
@@ -39,6 +39,8 @@ en:
       setting_up_storages: "For setting up file storages, please visit"
       setting_up_storages_non_admin: "Administrators can set up file storages in Administration / File Storages."
       all_available_storages_already_added: "All available storages are already added to the project."
+      oauth_client_id: "OAuth Client ID from attached storage (Nextcloud) that allows OpenProject to connect to the storage. This should be a lengthy string of garbled characters. Please see the storage manufacturers instructions on how to setup an OAuth application."
+      oauth_client_secret: "OAuth Client Secret from the attached storage (Nextcloud) above."
```

Comment:
I am not sure if that instruction here adds any value to the user. How does it help the user to know what to put in that field? How does it reduce doubt?

I don't have a better phrase. Shall we simple remove it?

Reply:
Meine Idee wäre es, die Nachricht durch einen einen Link auf ein Tutorial zu ergänzen, in dem für einen Admin aufgezeigt wird, wie das aufzusetzen ist. Das macht aber erst Sinn, wenn File-Links (einigermaßen) fertig ist. Vielleicht dann einen Field-Test mit 'nem unbedarften Admin?